app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user
+  
+  def create
+    @video = Video.find(params[:id])
+    @review = @video.reviews.build(params.require(:review).permit(:rating, :body))
+    @review.user = current_user
```

Comment:
I mean, when you assign current_user to `@review.user` you could rather have that merged into params.

`@review = @video.reviews.build(params.require(:review).permit(:rating, :body).merge!(user: current_user))` and yes I would personally prefer to push `params.require...` to its own private method - makes it more clear what is required and what is just merged in. Check instructors solution on how to merge if this would be an issue.